common/Jackdmp.cpp

```diff
@@ -253,8 +253,6 @@ void print_version()
     printf( "jackdmp version " VERSION " tmpdir "
             jack_server_dir " protocol %d" "\n",
             JACK_PROTOCOL_VERSION);
-    exit(-1);
```

Comment:
As all other functions that just print also do not exit the program, I moved this logic down to the `main()` body.